File: src/structures/phone.ts
import {Entity, PrimaryGeneratedColumn, Column, ManyToOne} from "typeorm";
import Costumer from "./costumer";

@Entity()
export default class Phone {

    @PrimaryGeneratedColumn()
    id: number;

    @Column({nullable:false})
    ddd: string;

    @Column({nullable:false})
    number: string;

    @ManyToOne(()=> Costumer, costumer => costumer.phones)
    costumer:Costumer;

}

File: Core/Models/Phone.cs
using System;
using System.ComponentModel.DataAnnotations.Schema;
using System.ComponentModel.DataAnnotations;
namespace Comies
{
    public class Phone : Entity
    {
        

        [MaxLength(2)]
        public string DDD { get; set; }

        [MaxLength(9)]
        public string Number { get; set; }
        public Guid? CostumerId { get; set; }
        public virtual Costumer Costumer { get; set; }
    }
}